packages/terra-application-header-layout/tests/wdio/application-header-spec.js

```diff
@@ -0,0 +1,15 @@
+/* global browser, Terra, beforeEach, expect */
+
+const viewports = Terra.viewports('tiny', 'medium');
```

Comment:
@dkasper-was-taken @tbiethman Does this need tested for a any particular viewport size?